tests/acceptance/config/behat.yml

```diff
@@ -119,7 +119,6 @@ default:
         - TUSContext:
         - SpacesTUSContext:
         - OcisConfigContext:
-        - TagContext:
```

Comment:
Please, check if we can remove `SearchContext` from `apiSpaces` and `apiGraph`

Reply:
not needed so removed `SearchContext`